src/main/java/fr/adrienbrault/idea/symfony2plugin/templating/PhpTemplateMissingInspection.java

```diff
@@ -39,7 +39,7 @@ public void visitElement(PsiElement element) {
 
     private void invoke(@NotNull ProblemsHolder holder, @NotNull PsiElement psiElement) {
         String templateNameIfMissing = getTemplateNameIfMissing(psiElement);
-        if(templateNameIfMissing == null) {
+        if(templateNameIfMissing == null || templateNameIfMissing.isEmpty()) {
```

Comment:
By design i ever avoid piping empty string to reduce this errors. Currently `fr.adrienbrault.idea.symfony2plugin.templating.PhpTemplateMissingInspection#getTemplateNameIfMissing` is already checking for a blank string. I think this error is more like to be on a invalid document structure at time of adding the inspection